demo/lightning/checkpoint/multinode/train.py

```diff
@@ -5,10 +5,9 @@
 from typing import Generator
 
 import torch
+import torch.distributed
```

Comment:
It doesnt seem like newly added code uses torch.distributed ? 

Also it would much clearer (aligns with the examples I see) to use
` import torch.distributed as dist`